components/annot-continuous-values.vue

```diff
@@ -8,7 +8,7 @@
 
             <v-select
                 data-cy="selectTransform"
-                :options="transformChoices"
+                :options="getTransformOptions(this.activeCategory)"
```

Comment:
Yes. This eliminates the need for a wrapper function. (Though since `options` now becomes non-reactive, it will break if we ever decide to allow the user to switch a column's/set of columns' category on the annotation page itself – a feature we've never discussed.)